util/cli.cr

```diff
@@ -5,6 +5,9 @@ require "http/client"
 require "colorize"
 require "option_parser"
 
+# A shard must have been built sometime in this span in order to be considered active.
+private TIME_WINDOW = (30 * 12).days # 12 months
```

Comment:
what do you think about more accurate calculations? Maybe using `1.year` or something.